tools/cli/admin.go

```diff
@@ -44,3 +44,35 @@ func newAdminWorkflowCommands() []cli.Command {
 		},
 	}
 }
+
+func newAdminHistoryHostCommands() []cli.Command {
+	return []cli.Command{
+		{
+			Name:    "describe",
+			Aliases: []string{"desc"},
+			Usage:   "Describe internal information of history host",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  FlagWorkflowIDWithAlias,
+					Usage: "WorkflowID",
+				},
+				cli.StringFlag{
+					Name:  FlagAddressWithAlias,
+					Usage: "Host address(IP:PORT)",
+				},
+				// We have to use string type here because the CLI framework will mix 0 and empty value
```

Comment:
You could still use IntFlag here. To tell difference between 0 and empty, use c.IsSet().

Reply:
Sounds good.